test/test_pebble.py

```diff
@@ -3599,3 +3601,42 @@ def recv():
     if hasattr(pytest, 'PytestUnhandledThreadExceptionWarning'):
         test_websocket_recv_raises = pytest.mark.filterwarnings(
             'ignore::pytest.PytestUnhandledThreadExceptionWarning')(test_websocket_recv_raises)
+
+
+class MyCharm(ops.CharmBase):
+    def __init__(self, framework: Framework):
+        super().__init__(framework)
```

Comment:
This comment will be moot given this will be moving, but this is essentially a no-op, so you could just use `pass` here, I think. Or, even terse, just avoid the class altogether and use `Harness(ops.CharmBase, ...)` directly -- I think that works.